Tests/BuildSystemIntegrationTests/CompilationDatabaseTests.swift

```diff
@@ -190,45 +190,12 @@ final class CompilationDatabaseTests: XCTestCase {
           "arguments": ["swiftc", "/a/a.swift"]
         }
       ]
-      """.write(to: tempDir.appendingPathComponent("compile_commands.json"), atomically: true, encoding: .utf8)
+      """.write(to: tempDir.appendingPathComponent(JSONCompilationDatabase.dbName), atomically: true, encoding: .utf8)
 
       XCTAssertNotNil(tryLoadCompilationDatabase(directory: tempDir))
     }
   }
 
-  func testJSONCompilationDatabaseFromCustomDirectory() async throws {
```

Comment:
There's a test for basically this in `WorkspaceTests` now (since `tryLoadCompilationDatabase` only loads from the directory itself). I'd be open to adding `projectRoot` tests if you wanted though.